docs/resources/new_in_doc.md

```diff
@@ -7,6 +7,45 @@ month_change: true
 
 This page contains recent highlights and notable changes in [[= product_name =]] documentation.
 
+## July 2025
+
+### v5.0.0
+
+[Ibexa DXP v5.0.0](https://doc.ibexa.co/en/5.0/release_notes/ibexa_dxp_v5.0/#ibexa-dxp-v500) is available,
+accompanied by [5.0 documentation](https://doc.ibexa.co/en/5.0/).
+Ibexa DXP 5.0 LTS benefits of the [Long-Term Support](https://www.ibexa.co/blog/continuous-innovation-how-ibexa-dxp-s-lifecycle-enhances-feature-delivery).
+
+See how to [update from 4.6 to 5.0](https://doc.ibexa.co/en/5.0/update_and_migration/from_4.6/update_to_5.0/).
+
+#### LTS Updates
+
+4.6's LTS Updates are features included out of the box in 5.0.
+The previous 4.6 installation instructions are, in 5.0, unneeded or replaced by configuration instructions.
+
+| 4.6                                                                                                              | 5.0                                                                                                |
+|:-----------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| [Install AI Actions](https://doc.ibexa.co/en/4.6/ai_actions/install_ai_actions/)                                 | [Configure AI Actions](https://doc.ibexa.co/en/5.0/ai_actions/configure_ai_actions/)               |
+| [Install Discounts](https://doc.ibexa.co/en/4.6/discounts/install_discounts/)                                    | [Customize Discounts](https://doc.ibexa.co/en/5.0/discounts/configure_discounts/) directly         |
+| [Date and time attributes: installation](https://doc.ibexa.co/en/4.6/pim/attributes/date_and_time/#installation) | [Date and time attributes: usage](https://doc.ibexa.co/en/5.0/pim/attributes/date_and_time/#usage) |
+| Not released yet                                                                                                 | [Symbol attribute](https://doc.ibexa.co/en/5.0/pim/attributes/symbol_attribute_type/)              |
+
+#### PHP API
+
+Documentation examples and [PHP API Reference](https://doc.ibexa.co/en/5.0/api/php_api/php_api_reference/) have been updated for 5.0.
+You can use [`ibexa/rector`](https://github.com/ibexa/rector#readme) package to help yourself maintain your custom PHP code quality.
+And consult the [Ibexa DXP v5.0 renames, deprecations and removals](https://doc.ibexa.co/en/5.0/release_notes/ibexa_dxp_v5.0_deprecations/).
```

Comment:
Please try to rewrite. Sentence sounds incomplete, shouldn't start with "And..."
```suggestion
And consult the [Ibexa DXP v5.0 renames, deprecations and removals](https://doc.ibexa.co/en/5.0/release_notes/ibexa_dxp_v5.0_deprecations/).
```

Reply:
@julitafalcondusza ?
```suggestion
Consult the [Ibexa DXP v5.0 renames, deprecations and removals](https://doc.ibexa.co/en/5.0/release_notes/ibexa_dxp_v5.0_deprecations/) to learn more about the changes.
```